pg-extend/src/lib.rs

```diff
@@ -60,17 +60,16 @@ macro_rules! pg_magic {
 }
 
 /// Returns the slice of Datums, and a parallel slice which specifies if the Datum passed in is (SQL) NULL
-pub fn get_args(
-    func_call_info: &pg_sys::FunctionCallInfoData,
-) -> (&[pg_sys::Datum], Vec<bool>) {
+pub fn get_args<'a>(
+    func_call_info: &'a pg_sys::FunctionCallInfoData,
+) -> (impl 'a + Iterator<Item=&pg_sys::Datum>, impl 'a + Iterator<Item=pg_bool::Bool>) {
```

Comment:
@stillinbeta and here we're now returning Impl Trait based iterators, bound to the lifetime of the FunctionCallInfo